README.md

```diff
@@ -61,4 +61,28 @@ $ make run
 ```
 
 You should be able to test the forum at [http://localhost:8090/dev/](http://localhost:8090/dev/).
-This is the development view that emulates a LTI consumer.
\ No newline at end of file
+This is the development view that emulates a LTI consumer.
+
+## Backoffice
```

Comment:
I would replace the term «backoffice» with «Django admin site» or «Django admin» since it seems to be its official name.

Reply:
thank you for your feedback I will do it